.github/workflows/validate.yml

```diff
@@ -0,0 +1,33 @@
+name: Validate
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  validate:
+    strategy:
+      matrix:
+        go-version: [1.14.x, 1.15.x, 1.16.x]
```

Comment:
Also testing with Go's 1.16 version. Won't remove version 1.14 until we also remove from the Prebid Server CI/CD

Reply:
Awesome. Updated